controllers/skatespotController.js

```diff
@@ -0,0 +1,55 @@
+/************
+ * DATABASE *
+ ************/
+
+var db = require('../models');
+
+// // GET /api/skatespots
```

Comment:
It seems like this code is mostly-working, but all commented out--I'm confused by the state this code was left in.